File: core/src/main/java/com/makeuseof/muocore/models/PublisherImage.java
package com.makeuseof.muocore.models;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * Created by abaikirov on 10/3/17.
 */

@JsonIgnoreProperties(ignoreUnknown = true)

public class PublisherImage {

    @JsonProperty("featured")
    public String featured;

    @JsonProperty("thumb")
    public String thumb;

    @JsonProperty("middle")
    public String middle;

}

File: app/src/main/kotlin/com/makeuseof/cryptocurrency/data/post/model/PublisherImage.kt
package com.makeuseof.cryptocurrency.data.post.model

import com.google.gson.annotations.SerializedName

/**
 * Created by askar on 11/19/18
 * with Android Studio
 */

data class PublisherImage(
        @SerializedName("featured") var featured: String?,
        @SerializedName("thumb") var thumb: String?,
        @SerializedName("middle") var middle: String?
)
